src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java

```diff
@@ -101,11 +100,10 @@ public void loggedInUserOnDesktopCanVoteForFirstReply() {
     firstReplyUpvoteButtonClickAddsAnUpvoteAndSecondClickRemovesTheUpvote();
   }
 
-  @Test(groups = "discussions-loggedInUserOnDesktopCanUpvote", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-loggedInUserOnDesktopCanUpvote")
   @Execute(asUser = User.USER_3)
-  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
-  public void loggedInUserOnDesktopCanUpvote() {
+  @InBrowser(browserSize = DESKTOP_RESOLUTION)
+  public void regularUserOnDesktopCanUpvote() {
```

Comment:
It's to keep names consistend with user access levels: http://mediawiki119.wikia.com/wiki/Help:User_access_levels

That's also the standard for test user docs: https://wikia-inc.atlassian.net/wiki/display/SOC/Manual+testing